.husky/pre-commit

```diff
@@ -4,4 +4,4 @@
 set -e  # die on error
 
 yarn prettier && npx lint-staged
-yarn turbo run extract-translations
+yarn run extract-translations
```

Comment:
This is the only repo using turbo in extract-translations, added the script in package.json using lerna, to conform with other repos.

CC: @denniskigen @ibacher

Reply:
We're intentionally using Turbo instead of Lerna. Please undo this (fine to do it via script, just use Turbo).

Reply:
Sure @ibacher !